Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -1096,18 +1096,18 @@
     count: 5 #50 heat damage if you hit all your shots, but wide spread
-    spread: 30
+    spread: 20
 
 - type: entity
-  name: narrow laser barrage
-  id: BulletLaserSpreadNarrow
-  parent: BulletLaser
+  name: slug laser barrage
+  id: BulletLaserSpreadSlug
+  parent: BulletLaserHeavy
   categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)